node/src/service.rs

```diff
@@ -609,8 +605,10 @@ where
 	Ok(BasicQueue::new(verifier, Box::new(block_import), None, &spawner, registry))
 }
 
+/// Uses the lookahead collator to support async backing.
+///
 /// Start an aura powered parachain node. Some system chains use this.
-pub async fn start_generic_aura_node(
+pub async fn start_generic_aura_lookahead_node(
```

Comment:
We no longer support the old `start_generic_aura_node` the new runtime is not compatible with it anymore.